explainers/beforematch.md

```diff
@@ -288,6 +307,50 @@ scroll after adding an async step.
 5. **content-visibility: hidden-matchable -> visible**: find-in-page scrolls to
 to the revealed text.
 
+### Alternatives
+Given the purpose of displaying `content-visibility: hidden-matchable` text
+when it is searched for, there are a number of alternatives we have considered.
+
+#### Automatic Unlocking
+`content-visibility: hidden-matchable` text would be automatically changed to
+`content-visibility: visible` when the hidden text is searched for.
+##### Pros
+* Provides the desired behavior without the need for the web developer to do
+  anything besides use the `hidden-matchable` value.
+* Less complex than `beforematch` with regards to scrolling behavior.
+##### Cons
+* Doesn't allow the developer to change other properties in conjunction with
+  displaying hidden content. For example, in "Example 1: Expanding
+  `hidden-matchable`," automatic unlocking wouldn't be able to change the
+  style of the `<h1>` title elements.
+
+#### Automatic Unlocking with Activation Event
+This is like "Automatic Unlocking," but with an added "activation" emitted when
+content is displayed to allow developers to change other state and styles if
+needed.
+##### Pros
+* Same pros as "Automatic Unlocking."
+* Allows other state/styles to be changed when content is displayed.
+##### Cons
+* TODO: ask vmpstr and chrishtr for more context on this
+
+#### CSS Pseudo Selector
+A pseudo selector, such as `:target`, would be applied to the element
+containing the matching text when it is searched for.
+##### Pros
+* Allows content to become visible when searched for with only CSS.
+##### Cons
+* If CSS with a pseudo selector is used to make text visible, then when
+  find-in-page is closed or the search text changes, the pseudo selector would
+  be removed and then any selector which is displaying the text based on that
+  pseudo selector would not apply, causing the expanded section to unexpectedly
+  collapse.
+* There is no way in CSS to say "change my style if a descendant has a pseudo
+  class on it." For this reason, developers would be unable to change styles
+  outside of the particular element that has the matching text, which would
```

Comment:
This isn't necessarily true if we add the pseudo class on the ancestor hierarchy. I think the point here is that it can be complicated or impossible to provide the right selector which can modify a style on an unrelated element (Such as the case in the title-collapsed or title-expanded that you have above

Reply:
Thanks for the explanation, I put your words into this bullet point